core/templates/pages/about-page/about-page.directive.html

```diff
@@ -103,435 +103,9 @@ <h2 translate="I18N_ABOUT_PAGE_CREDITS_TAB_HEADING"></h2>
-                <li>Sourab Jha</li>
-                <li>Sourav Badami</li>
-                <li>Sourav Singh</li>
-                <li>Sreenivasulu Giritheja</li>
-                <li>Srijan Reddy</li>
-                <li>Srikar Ch</li>
-                <li>Stefanie Muroya Lei</li>
-                <li>Stephanie Federwisch</li>
-                <li>Stephen Chiang</li>
-                <li>Sudhanva MG</li>
-                <li>Sudipta Gyan Prakash Pradhan</li>
-                <li>Sumit Paroothi</li>
-              </ul>
-
-              <span>T</span>
-              <ul>
-                <li>Tanmay Mathur</li>
-                <li>Tarashish Mishra</li>
-                <li>Teddy Marchildon</li>
-                <li>Tezuesh Varshney</li>
-                <li>Timothy Cyrus</li>
-                <li>Tonatiuh Garcia</li>
-                <li>Tony Afula</li>
-                <li>Tony Jiang</li>
-                <li>Tracy Homer</li>
-                <li>Travis Shafer</li>
-                <li>Truong Kim</li>
-                <li>Tuguldur Baigalmaa</li>
-                <li>Tyler Ishikawa</li>
-              </ul>
-
-              <span>U</span>
-              <ul>
-                <li>Ujjwal Gulecha</li>
-                <li>Umesh Singla</li>
-                <li>Utkarsh Dixit</li>
-              </ul>
-
-              <span>V</span>
-              <ul>
-                <li>Varun Tandon</li>
-                <li>Vasu Tomar</li>
-                <li>Vibhor Agarwal</li>
-                <li>Viet Tran Quoc Hoang</li>
-                <li>Viktor Pishuk</li>
-                <li>Vinit Dantkale</li>
-                <li>Vinita Murthi</li>
-                <li>Viraj Prabhu</li>
-                <li>Vishal Desai</li>
-                <li>Vishal Gupta</li>
-                <li>Vishal Joisar</li>
-                <li>Vishnu M</li>
-                <li>Vojtěch Jelínek</li>
-                <li>Vuyisile Ndlovu</li>
-              </ul>
-
-              <span>W</span>
-              <ul>
-                <li>Will Li</li>
-                <li>Wilson Hong</li>
-              </ul>
-
-              <span>X</span>
-              <ul>
-                <li>Xinyu Wu</li>
-                <li>Xuân (Sean) Lương</li>
-              </ul>
-
-              <span>Y</span>
-              <ul>
-                <li>Yana Malysheva</li>
-                <li>Yang Lu</li>
-                <li>Yash Jipkate</li>
-                <li>Yash Ladha</li>
-                <li>Yi Yan</li>
-                <li>Yiming Pan</li>
-                <li>Yogesh Sharma</li>
-                <li>Yousef Hamza</li>
-                <li>Yuan Gu</li>
-                <li>Yuliang</li>
-              </ul>
-
-              <span>Z</span>
-              <ul>
-                <li>Zach Puller</li>
-                <li>Zachery Vekovius</li>
-                <li>Zoe Madden-Wood</li>
+              <span ng-repeat-start="alphabet in $ctrl.ALPHABETS" ng-if="$ctrl.hasCredits(alphabet)">{{alphabet}}</span>
```

Comment:
I'm a bit surprised that this works. Shouldn't we be using angle brackets for interpolation: `<[...]>`, as we do everywhere else? Does everything show up correctly when you view the page?

Also /cc @vojtechjelinek since this might be some artifact of Jinja.

Reply:
I also recently discovered this and fixed a lot of occurrences in #9201. This is not Jinja, it is evaluated via Angular(JS) I'm not sure why both `<[]>` and `{{}}` work.

Reply:
Yes, both of them work but I have updated now to `<[]>`.

Reply:
@vojtechjelinek @seanlip should we add a [BAD_PATTERN] lint test for this?

Reply:
Yep, will you create an issue for this please?